incent-config/Dangerfile

```diff
@@ -89,19 +89,19 @@ def commit_body
 end
 
 def not_applicable_pr_title?
-  github.pr_title.start_with?('[na]')
+  github.pr_title.start_with?('[na]') || github.pr_title.include?('(na)')
 end
 
 def not_applicable_branch_name?
   branch_name.include?('/na/')
 end
 
 def not_applicable_subject?
-  commit_subject.include?('[na]')
+  commit_subject.include?('[na]') || commit_subject.include?('(na)')
 end
 
 def not_applicable_body?
-  commit_body.include?('[na]')
+  commit_body.include?('[na]') || commit_body.include?('(na)')
```

Comment:
Possible refactor here (extracting this logic). Also, maybe the extracted method could have logic along these lines (if you think it's readable).

```rb
["[na]", "(na)"].any? { |token| "foo".include?(token) }
```